app/routes/_index.tsx

```diff
@@ -26,21 +78,202 @@ export const loader: LoaderFunction = async () => {
+						<p className="text-3xl md:text-3xl font-bold">
+							3.6K <span className="text-lg">USD</span>
+						</p>
+					</div>
+				</div>
+				<div className="flex flex-auto items-center gap-4 lg:w-[33%] rounded-3xl bg-vd-blue-200 p-4">
+					<img src={"/blue_candle.svg"} alt="blue candle drawing" />
+					<div className="flex flex-col gap-2">
+						<p className="text-base font-medium"># of Reports Fully Funded</p>
+						<p className="text-3xl md:text-3xl font-bold">12</p>
+					</div>
+				</div>
+			</section>
+
+			<article className="w-full max-w-[1372px]">
+				<h2 className="text-3xl md:text-4xl font-semibold pt-6">Reports</h2>
+				<div className="flex flex-col md:flex-row md:justify-between md:items-end pb-8">
+					<p className="text-base pb-4 md:pb-0 ">
+						Find and fund reports that resonate with you.
+					</p>
+					<div className="flex flex-col md:flex-row gap-3">
+						<Input type="search" placeholder="Search Reports" />
+						<Select>
+							<SelectTrigger>
+								<SelectValue placeholder="Sort by" />
+							</SelectTrigger>
+							<SelectContent>
+								<SelectItem value="amount-needed">Amount Needed</SelectItem>
+								<SelectItem value="newest-oldest">Newest to Oldest</SelectItem>
+								<SelectItem value="oldest-newest">Oldest to Newest</SelectItem>
+								<SelectItem value="most-contributors">
+									Most Contributors
+								</SelectItem>
+							</SelectContent>
+						</Select>
+					</div>
+				</div>
+
+				<div className="flex flex-col md:flex-row gap-10 pb-16">
+					<section>
+						<div className="border border-b-vd-blue-400 py-4">
+							<h2 className="text-base font-medium pb-4">Categories</h2>
+							{Array.from(
+								new Set(reports.map((report: Report) => report.category)),
+							).map((category) => (
+								<div className="flex items-center gap-2 pb-1">
+									{GetIcon({
+										category: category as string,
+										color: "#E48F85",
+										strokeWidth: "1.5",
+										size: "26",
+									})}
+									<p className="text-sm">{category as string}</p>
+								</div>
+							))}
+						</div>
+						<div className="border border-b-vd-blue-400 pt-6 pb-4">
+							<h2 className="text-base font-medium pb-4">Amount needed</h2>
+							{Array.from(
+								new Set(reports.map((report: Report) => report.fundedSoFar)),
+							).map((fundedSoFar) => (
+								<div className="flex items-center gap-2 pb-1">
+									<Circle size={18} strokeWidth={1} />
+									<p className="text-sm">${1000 - (fundedSoFar as number)}</p>
+								</div>
+							))}
+						</div>
+						<div className="border border-b-vd-blue-400 pt-6 pb-4">
+							<h2 className="text-base font-medium pb-4">Story from</h2>
+							{Array.from(
+								new Set(reports.map((report: Report) => report.id)),
+							).map((id) => (
+								<div className="flex items-center gap-2 pb-1">
+									<Circle size={18} strokeWidth={1} />
+									<p className="text-sm">{(id as string).slice(0, 15)}</p>
```

Comment:
Lots of Typecasting going on. Let's think on this? Kinda tricky with the Set and Array getting created.

Reply:
trying .filter instead of `Array.from(new Set(reports.map....))` to avoid the typecasting, wdyt?

Reply:
Great call. Filter, reduce or even map would probably all work great.  I think filter would probably be the cleanest and is descriptive of what the intention is.

Reduce and Map might have a slight edge in performance with very large data sets but I don't think we're really worried about that.